modules/jaggery-apps/app-publisher-web/apis/v1/assets_api_router.jag

```diff
@@ -30,7 +30,7 @@ require('/modules/publisher.js').exec(function (ctx) {
         }
 
 
```

Comment:
There are many formatting issues in this jag file which is not relevant to this PR and also this formatting issue is not relevant to this PR. So rise a public jira to format whole jag file. - https://wso2.org/jira/browse/APPM-1251
And also this file git repo had been changed to the carbon-appmgt. https://github.com/wso2/carbon-appmgt/blob/master/features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/apis/v1/assets_api_router.jag